site2/docs/client-libraries-cpp.md

```diff
@@ -65,7 +124,7 @@ by using user local system provided version is more reasonable, and more easy fo
 $ rpm -ivh apache-pulsar-client*.rpm
 ```
 
-After install, Pulsar libraries will be placed under `/usr/lib`.
+With a successful installation, you will see Pulsar libraries under `/usr/lib`.
```

Comment:
After you install RPM successfully, Pulsar libraries are in the `/usr/lib` path.